### PR TITLE
Fix for ru.wikipedia.org

### PR DESCRIPTION
**Pull request by Rakleed**
Example: https://ru.wikipedia.org/wiki/Заглавная_страница

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4664,6 +4664,9 @@ a[href$=".svg"]:hover > img {
         background-color: ${#eaecf0} !important;
     }
 }
+.main-top {
+    background: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4668]: Why removing the background of this?

**Comment by Rakleed** [comment]
<details>
<summary>Before</summary>

screenshot-ru-wikipedia-org-wiki-D0-97-D0-B0-D0-B3-D0-BB-D0-B0-D0-B2-D0-BD--1592483755177

</details>

<details>
<summary>After</summary>

screenshot-ru-wikipedia-org-wiki-D0-97-D0-B0-D0-B3-D0-BB-D0-B0-D0-B2-D0-BD--1592483796070

</details>

**Comment by Gusted** [comment]
Side-note: Do you use a special program to make screenshot? I see some kind of `shadow fade` outside the screenshot itself.

Seems good.

**Comment by Rakleed** [comment]
> Side-note: Do you use a special program to make screenshot? I see some kind of `shadow fade` outside the screenshot itself.

I am using the [Standardized Screenshot extension](https://chrome.google.com/webstore/detail/standardized-screenshot/pabdhaakclnechgfhmnhkcbmjobeoope) for Chrome.

**Pull request merged**